lib/bulk-apply-applications.js

```diff
@@ -243,91 +261,166 @@ function bulkApplyApplications (args, callback) {
+                if (err) {
+                  return err;
                 } else {
-                  payload.to = parent.email;
-                  if (!_.contains(parentsEmailed, payload.to) && payload.to !== profile.email) {
-                    parentsEmailed.push(payload.to);
-                    seneca.act({role: 'cd-dojos', cmd: 'send_email', payload: payload}, cb);
-                  } else {
-                    return cb();
-                  }
+                  return res;
                 }
-              }, cb);
+              })
+            });
+          }
+        },
+        //send email to parents about all applied children
+        function sendParentsEmail () {
+          var parentName, parentEmail;
+          var childrensNames = [];
+          emailSubject = parentEmailSubjectData[applications[0].status];
+          async.each(profiles, function(profile, profileCb) {
+            //if this profile has parents, it is a child
+            if (!_.isEmpty(profile.parents)) {
+              childrensNames.push(profile.name);
+              parentName = profile.parents[0].name;
+              parentEmail = profile.parents[0].email;
             }
+            profileCb();
           });
-        }
-      });
-      return done(null, applications);
+          //set email intro based on application status
+          if (applications[0].status === 'pending') {
+            emailIntro = i18nHelper.getClosestTranslation(locality, 'This is a notification to let you know that a request for a ticket for the below event has been received for your child %1s. Once the request has been approved they will receive their ticket confirmation by email.');
+            emailIntro = emailIntro.ifPlural(childrensNames.length, 'This is a notification to let you know that requests for tickets for the below event have been received for your children %1s. Once the requests has been approved they will receive their ticket confirmations by email.').fetch([childrensNames.join(', ')]);
+          } else if (applications[0].status === 'approved') {
+            emailIntro = i18nHelper.getClosestTranslation(locality, 'This is an order confirmation for your child %1s for the below event.');
+            emailIntro = emailIntro.ifPlural(childrensNames.length, 'This is an order confirmation for your children %1s for the below event.').fetch([childrensNames.join(', ')]);
+          } else if (applications[0].status === 'cancelled') {
+            emailIntro = i18nHelper.getClosestTranslation(locality, 'A ticket for your child %1s for the below event has been cancelled.');
+            emailIntro = emailIntro.ifPlural(childrensNames.length, 'Tickets for your children %1s for the below event have been cancelled.').fetch([childrensNames.join(', ')]);
+          }
+          //email payload changes for parents
+          var payloadChanges = {
+            to: parentEmail || null,
```

Comment:
In reference to previous comment regardign multiple parents, considering bulk-apply is family restricted anyway (there are no action to my knowledge that applies to multiple families at the same time), parentEmails could be a joined list. Not sure if it works, but worth a try.